src/test/kotlin/chapter15/exercises/ex1/listing.kt

```diff
@@ -34,15 +34,15 @@ fun <I> dropWhile(p: (I) -> Boolean): Process<I, I> =
 
 //TODO: Enable tests by removing `!` prefix
 class Exercise1 : WordSpec({
-    val stream = Stream.of(1, 2, 3, 4, 5, 6, 7)
+    val stream = Stream.of(1, 2, 3, 4, 5, 6, 7, 1)
```

Comment:
All the tests seem to be testing the boundary of 5 elements. How about we ditch the magic number 1 and make the sequence something like `1, 2, 3, 4, 5, 6, 5, 4, 3, 2, 1`?

Reply:
Yes, sure. I will update the class in this branch and replace the merge request.

Reply:
It's all done, and GitHub seems to take my new commit into account in this same merge request. I suggest double-checking this when you merge it into the master branch.